pkg/controller/networkpolicy/validate.go

```diff
@@ -615,6 +624,78 @@ func (v *antreaPolicyValidator) validateTierForPassAction(tier string, ingress,
 	return "", true
 }
 
+func (v *antreaPolicyValidator) validateEgressMulticastAddress(egressRule []crdv1alpha1.Rule) (string, bool) {
+	for _, r := range egressRule {
+		multicast := false
+		unicast := false
+		otherSelectors := false
+		for _, to := range r.To {
+			toIPAddr, _, err := net.ParseCIDR(to.IPBlock.CIDR)
+			if err != nil {
+				return fmt.Sprintf("invalid multicast ip address (to.IPBlock.CIDR): %v", err.Error()), false
+			}
+			if toIPAddr.IsMulticast() {
+				multicast = true
+			} else{
+				unicast = true
+			}
+			if to.PodSelector != nil || to.NamespaceSelector != nil || to.Namespaces != nil ||
+				to.ExternalEntitySelector != nil || to.ServiceAccount != nil || to.NodeSelector != nil {
+				otherSelectors = true
+			}
+			if multicast && unicast {
+				return fmt.Sprintf("can not set multicast ip address and unicast ip address at the same time"), false
```

Comment:
if user set both unicast and multicast address, then the rule has to write the tables both for unicast and multicast, it will make the logic too compilicate

Reply:
done